### PR TITLE
Add ppafm-conv-rho and ppafm-generate-dftd3. Closes #186

### LINKED ISSUE #186: Merge `relaxed_scan_PVE.py` into `relaxed_scan.py`?
While doing #185, I noted that the `relaxed_scan_PVE.py` script used in the density overlap model is almost identical to the standard `relaxed_scan.py` except for a couple of additions.

@ProkopHapala Do you think we should merge these two scripts into one? It seems like this could be done quite easily just by adding some argument like `--noLJ` to `relaxed_scan.py`.

### PR DESCRIPTION
**Pull request by mondracek**
Scripts `ppafm-conv-rho` and `ppafm-generate-dftd3` can be used to call `conv_rho.py` and `generateDFTD3.py`, respectively, now. The script `example/pyridineDensOverlap/run.sh` was updated. Problem of undefined `geometry_format` in `getAtomsWhichTouchPBCcell` resolved as testing the `pyridineDensOverlap` example. Done in the aftermath of and should close #186.

### CODE REVIEW

File: ppafm/cli/conv_rho.py
Comment (ProkopHapala) [inline ppafm/cli/conv_rho.py:25]: So the intend here is to be able call this as a function from another script? Or what is the purpose/advantage to encapsulate everything inside main function?

**Review by ProkopHapala — APPROVED** [review]
I assume it works. Otherwise I don't see much for dicussion. Thanks.

File: ppafm/cli/conv_rho.py
Comment (mondracek) [reply ppafm/cli/conv_rho.py:25]: I do not much about how exactly this works, I just followed the template indicated by @NikoOinonen and used e.g. for `ppafm-generate-elff`, but... if you look inside the script `ppafm-conv-rho`, which gets installed by `pip` based on `pyproject.toml`, this script actually loads `conv_rho.py` as ``from ppafm.cli.conv_rho import main`` and then calls it by ``sys.exit(main())``. I have no idea if it could be done in a different way.

Comment (NikoOinonen) [reply ppafm/cli/conv_rho.py:25]: There is a feature in setuptools called [_entry point_](https://setuptools.pypa.io/en/latest/userguide/entry_point.html), which allows easily adding executable scripts to the package distribution. The idea is that you define a function somewhere in your package, and then you automatically generate an installable script that simply imports and calls that function.

File: ppafm/cli/conv_rho.py
Comment (NikoOinonen) [inline ppafm/cli/conv_rho.py:13]: I don't think this is needed. You can get a help message for the script by simply calling it with the `-h` option. E.g. `ppafm-conv-rho -h`.

**Comment by NikoOinonen** [comment]
I checked that it works. Naturally it requires a reinstall for the new scripts.

File: ppafm/cli/conv_rho.py
Comment (mondracek) [reply ppafm/cli/conv_rho.py:13]: Alright. However, `generateElFF.py` does have the `HELP_MESSAGE` constant defined like the above. Is there any reason why `generateElFF.py` is special? Or should I remove the `HELP_MESSAGE`  from `generateElFF.py` too?

Comment (NikoOinonen) [reply ppafm/cli/conv_rho.py:13]: I don't see any reason why `generateElFF.py` would need it either. If you need to print the help message from within the program, you can use the [`parser.print_help()`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.print_help) method. I would remove this from both of the scripts.

**Comment by NikoOinonen** [comment]
@mondracek I guess we can merge this now, if you don't have any other concerns? I think we should also make a new release after this is merged, so that the examples work correctly on the release version.

**Comment by mondracek** [comment]
Good, squashing and merging `alt-relaxed-scan` into `main`. Regarding the new version, should it be `v0.2.2` or something else? Anyway, I will leave it up to @NikoOinonen or others to make the new release version.

**Comment by NikoOinonen** [comment]
New release 0.2.2 is out: https://github.com/Probe-Particle/ppafm/releases/tag/v0.2.2

**Pull request merged**
